docs/examples/agent/openai_agent_query_cookbook.ipynb

```diff
@@ -67,7 +67,23 @@
     "import os\n",
```

Comment:
<p>I don't think this is actually needed, pyndantic will do this for you under the hood</p><p> </p><pre class="ql-syntax" spellcheck="false">&gt;&gt;&gt; filter = MetadataFilter(key="test", value=1, operator="&gt;=")
&gt;&gt;&gt; filter = MetadataFilter(key="test", value=1, operator="&gt;==")
Traceback (most recent call last):
&nbsp; File "&lt;stdin&gt;", line 1, in &lt;module&gt;
&nbsp; File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for MetadataFilter
operator
&nbsp; value is not a valid enumeration member; permitted: '==', '&gt;', '&lt;', '!=', '&gt;=', '&lt;=' (type=type_error.enum; enum_values=[&lt;FilterOperator.EQ: '=='&gt;, &lt;FilterOperator.GT: '&gt;'&gt;, &lt;FilterOperator.LT: '&lt;'&gt;, &lt;FilterOperator.NE: '!='&gt;, &lt;FilterOperator.GTE: '&gt;='&gt;, &lt;FilterOperator.LTE: '&lt;='&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>